dist/README.md

```diff
@@ -0,0 +1,30 @@
+# Dist
+
+The `dist` folder contains sample configs for various platforms.
+
+## Conventions
+
+On unixes we will place agent configs under `/etc/nomad` and store data under `/var/lib/nomad/`. You will need to create both of these directories. We assume that `nomad` is installed to `/usr/local/bin/nomad`.
```

Comment:
If we are assuming nomad is installed in /usr/local/bin, then I think having the data dir in /usr/local/var/lib/nomad makes more sense?

/var/lib/nomad would have made more sense if nomad was installed in /bin/nomad. Not a big deal either ways.